api/src/main/java/org/openmrs/api/impl/PatientServiceImpl.java

```diff
@@ -1318,8 +1318,7 @@ public IdentifierValidator getIdentifierValidator(String pivClassName) {
 			return getIdentifierValidator((Class<IdentifierValidator>) Context.loadClass(pivClassName));
 		}
 		catch (ClassNotFoundException e) {
-			log.error("Could not find patient identifier validator " + pivClassName, e);
-			return getDefaultIdentifierValidator();
+			throw new PatientIdentifierException("Could not find patient identifier validator " + pivClassName);
```

Comment:
Can we also pass the original exception in the constructor?

Reply:
passed the original exception to the constructor.